src/traits/mod.rs

```diff
@@ -41,8 +41,7 @@ pub trait Group:
     + for<'de> Deserialize<'de>;
 
   /// A type representing an element of the scalar field of the group
-  type Scalar: PrimeField
```

Comment:
@huitseeker this removed line is necessary, right?

Reply:
No, otherwise this would not compile. This is implied by `PrimeFieldBits`: https://github.com/zkcrypto/ff/blob/e853770f4843e47cd5e07330140b5bd1cdfc9f11/src/lib.rs#L453